manifests/supervisorcluster/1.29/cns-csi.yaml

```diff
@@ -160,6 +160,9 @@ rules:
   - apiGroups: [""]
     resources: ["persistentvolumes"]
     verbs: ["get", "list", "update", "delete"]
+  - apiGroups: ["cns.vmware.com"]
+    resources: ["cnsfileaccessconfigs"]
+    verbs: ["get", "list", "create"]
```

Comment:
PR title says add RBAC rule to allow DevOps user to Create, delete and List cnsfileaccessconfigs.

Are you adding delete verb in the RBAC?

Reply:
I am adding delete privilege also.

Changed the title to reflect all changes.